static/src/javascripts/projects/commercial/modules/hosted/onward.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import fetchJson from 'lib/fetch-json';
+import fastdom from 'lib/fastdom-promise';
+import HostedCarousel from 'commercial/modules/hosted/onward-journey-carousel';
+
+const loadOnwardComponent = (
+    start: () => void,
+    stop: () => void
+): Promise<any> => {
+    start();
+
+    const placeholders = document.getElementsByClassName(
+        'js-onward-placeholder'
+    );
+
+    if (placeholders.length) {
+        fetchJson(
+            `${config.page.ajaxUrl}/${config.page.pageId}/${config.page.contentType.toLowerCase()}/` +
+                `onward.json`,
+            {
+                mode: 'cors',
+            }
+        )
+            .then(json =>
+                fastdom.write(() => {
+                    let i;
+                    for (i = 0; i < placeholders.length; i += 1) {
```

Comment:
> Unlike arrays, HTMLCollections do not have a forEach method 😞

Ah sorry, I didn't read the whole context of this module, I just saw the for loop 😄 

We can `Array.from` an iterable in js to make them behave like arrays:

```js
Array.from(placeholders).forEach(el => {
  // el is a HTMLElement
});
```

This will avoid the need to do `if (placeholders.length)` too.